security/src/main/java/zone/cogni/lib/security/permission/PermissionService.java

```diff
@@ -27,33 +30,97 @@ public class PermissionService {
   private final Map<String, Set<String>> roleName2permissionString = Collections.synchronizedMap(new HashMap<>());
   private final Set<String> defaultPermissionStrings;
   private final Map<String, Permission> permissionByName;
+  private final Map<String, String> ldap2roleName = new HashMap<>();
+  private final Map<String, Set<String>> blacklistProperties = new HashMap<>();
+  private final Set<String> allBlacklistProperties;
+  private final Map<String, Set<String>> notBlacklistedForRole = new HashMap<>();
+
+  @Value("${lux.ldap.role.group.resource:/ldap-role-group.json}")
+  private String ldapRoleGroupResource;
```

Comment:
security lib is trying to be generic
this is all lux specific stuff, should not be here

for sure property `lux.ldap.role.group.resource` tells you it's not ok